httpbakery/agent/legacy_test.go

```diff
@@ -301,8 +280,9 @@ func (s *legacyAgentSuite) visit(p httprequest.Params, dischargeId string, rende
 // LegacyAgentHandler represents a handler for legacy
 // agent interactions. Each member corresponds to an HTTP endpoint,
 type LegacyAgentHandler struct {
-	Visit func(p httprequest.Params, dischargeId string) error
-	Wait  func(p httprequest.Params, dischargeId string) (*bakery.Macaroon, error)
+	VisitGet  func(p httprequest.Params, dischargeId string) error
+	AgentPost agentPostFunc
```

Comment:
Is it OK to have an exported field with an un-exported type? This probably needs a comment

Reply:
it is ok, but generally a bit frowned on. i'll unexport the fields - it's all test-local anyway.